lib/runner.js

```diff
@@ -72,27 +72,31 @@ runner.run = async () => {
 	/**
 	 * Process an aXe issue.
 	 * @private
-	 * @param {Object} issue - An aXe issue to process.
-	 * @returns {Object} Returns the processed issue.
+	 * @param {Object} axeIssue - An aXe issue to process.
+	 * @returns {Object[]} Returns an array of processed issues.
 	 */
-	function processIssue(issue) {
+	function processIssue(axeIssue) {
+		// See https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#results-object for a description
+		// of an axe issue
 
-		// For now we just select the first element. This needs testing
-		const selector = issue.nodes.map(node => selectorToString(node.target)).join(', ');
-		const element = (issue.nodes.length ? window.document.querySelector(selector) : null);
+		let elements = [null];
```

Comment:
How's test coverage when `axeIssues.nodes.length` is zero? Will this do what we want?

Reply:
There is an entry in the test with `id: 'mock-id-no-nodes'` for this line.

Reply:
NYC is reporting 100% coverage